src/GatewayMethod.php

```diff
@@ -1601,4 +1574,39 @@ public static function versionCheck(string $version = '3.0'): bool
 
         return false;
     }
+
+    private function getCountryEnvironment()
+    {
+        $defaultEnvironments = [
```

Comment:
Reduce vars usage: remove $defaultEnvironments and use array in return with array_merge, not? Rename $environment to $environments?